changes/12.added

```diff
@@ -0,0 +1 @@
+Added the ability to display Tenant and TenantGroup of a Rack on Tiles and views.
```

Comment:
> Added the ability to display

This wording makes it sounds configurable, but the changes seem to be on by default. Did you mean to add a toggle for this that the end user can enable/disable?

Reply:
Currently it is on by default. It would be great to make it a toggle feature, but I think we should make changes to svg.py before we do that. Right now the text that is displayed is in a fixed position based on type (rack, rackgroup, tenant, etc). I think it would make sense and look cleaner to have the location adjusted based off of the user defined fields that are displayed.

Reply:
```suggestion
Added a Rack's Tenant (and TenantGroup if applicable) to Tiles and detail views.
```